src/__tests__/actions/resetAction/__test__/changePasswordActions.test.js

```diff
@@ -0,0 +1,73 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import fetchMock from "fetch-mock";
+import changePasswordAction from "../../../../actions/resetAction/changePasswordActions";
+import { RESET_PASSWORD_SUCCESS, RESET_PASSWORD_FAIL } from "../../../../actions/types";
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const { BASE_URL } = process.env;
+
+
```

Comment:
@Peace-Apple Please clear these blank-lines for more readability.

Reply:
Thanks Joel let me fix them

Reply:
I have fixed them now @joelethan